<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Members</class>
 <widget class="QDialog" name="Members">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addMembersButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Members</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteMembersButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>390</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Members</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Member</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Member Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Is Borrowing</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>001</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>Colin</string>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>002</string>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string>Luke</string>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string>003</string>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string>Bob</string>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>28</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Member Management</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
